backend/codesirius/authentication/models/verification.py

```
@@ -93,6 +93,7 @@ def save(self, *args, **kwargs):
 
         try:
             # Send the verification code to the user
+            logger.info(f"Sending verification code to {self.user.email}")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)